src/core/Baked.Architecture/Bake.cs

```diff
@@ -4,9 +4,15 @@
 
 namespace Baked;
 
-public class Bake(IBanner _banner, Func<Application> _newApplication)
+public class Bake(IBanner _banner, Func<Application> _newApplication,
+    bool _bake = false,
+    bool _start = true
+)
 {
-    public static Bake New => new(new BakedBanner(), () => new(new()));
+    public static Bake New => new(new BakedBanner(), () => new(new()),
+        _bake: Environment.GetCommandLineArgs().Any(a => a.EndsWith("bake")),
+        _start: !Environment.GetCommandLineArgs().Any(a => a.EndsWith("no-start"))
```

Comment:
don't forget to remove this temp solution. it should explicitly look for `--bake` and `--no-start` args, not anything that ends with `bake` and `no-start`